sequence_processing_pipeline/templates/nuqc_job.sh

```diff
@@ -92,13 +87,15 @@ do
 
     # for now just make sure each file is saved and we can read the data inside
     # to sort them out later.
-    html_name=$(echo "$r1_name.html")
-    json_name=$(echo "$r1_name.json")
+
+    s_name=$(basename "${r1}" | sed -r 's/\.fastq\.gz//')
+    html_name=$(echo "$s_name.html")
+    json_name=$(echo "$s_name.json")
 
     echo "${i}	${r1_name}	${r2_name}	${base}" >> ${TMPDIR}/id_map
 
     fastp \
-        -l 45 \
+        -l 100 \
         -i ${r1} \
         -I ${r2} \
         -w 7 \
```

Comment:
should this be {{cores_per_task}} ? Same for minimap2 threads?

Reply:
sure I can make it a param.